core/src/main/java/org/infinispan/configuration/global/GlobalConfigurationBuilder.java

```diff
@@ -157,6 +161,11 @@ public ShutdownConfigurationBuilder shutdown() {
       return modules;
    }
 
+   public GlobalConfigurationBuilder zeroCapacityNode(boolean zeroCapacityNode) {
```

Comment:
Seems to be missing some documentation. Also I don't remember the discussions, but is this going to be fully supported mode? I wonder if we should add an @Experimental tag to it?

Reply:
what do you mean by fully supported?

Reply:
I am saying as in it doesn't have @Experimental tag on it and is shown to users and properly documented everything. For example this method has no javadoc on it, doesn't tell me what this means at all.

Reply:
@wburns I added some extra javadoc in a commit

Reply:
@wburns some java doc added in another commit